### PR TITLE
Update `docker-compose.yml`

### PR DESCRIPTION
**Pull request by slashtechno**
Updated `docker-compose.yml` to use the image for TriliumNext, in addition to adding some more comments.

Related to [triliumnext/Docs#102](https://github.com/TriliumNext/Docs/pull/102)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,16 +1,23 @@
+# TL;DR:
```
Comment (rauenzi) [inline docker-compose.yml:1]: This feel unneeded, remove

**Review by rauenzi — COMMENTED** [review]
Why the change in spacing? I think 2 is well suited for yaml and is what is already being used.

Just my opinion though, I'll let Elian decide.

**Comment by eliandoran** [comment]
> Why the change in spacing? I think 2 is well suited for yaml and is what is already being used.

@rauenzi , I think it might be caused by the editorconfig we have, I think I made a similar accidental change on my side when messing with the Docker build.

@slashtechno , if it would be possible for you to revert the change in spacing, it would be appreciated since it makes for a cleaner diff.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
-        volumes:
-            # Unless TRILIUM_DATA_DIR is set, the data will be stored in the "trilium-data" directory in the home directory.
-            # This can also be changed with by replacing the line below with `- /path/of/your/choice:/home/node/trilium-data
-            - ${TRILIUM_DATA_DIR:-~/trilium-data}:/home/node/trilium-data
+  trilium:
+    # Optionally, replace `latest` with a version tag like `v0.90.3`
+    # Using `latest` may cause unintended updates to the container
+    image: triliumnext/notes:latest
+    # Restart the container unless it was stopped by the user
+    restart: unless-stopped`
```
Comment (rauenzi) [inline docker-compose.yml:10]: Erroneous backtick?

**Review by chesspro13 — APPROVED** [review]
Looks good to me.

**Review by eliandoran — APPROVED** [review]
LGTM.

**Pull request merged**